src/caikit_nlp_client/http_client.py

```diff
@@ -58,17 +58,19 @@ def __init__(
                 (
                     self._client_key_path,
                     self._client_cert_path,
+                    self._ca_cert_path,
                 )
             )
             and not self._mtls_configured
         ):
             raise ValueError(
-                "Must provide both client_cert_path and client_key_path for mTLS"
+                "Must provide both ca_cert_path, client_cert_path, \
+                    client_key_path for mTLS"
```

Comment:
small nitpick 

```suggestion
                "Must provide ca_cert_path, client_cert_path and client_key_path for mTLS"
```

Reply:
I had that originally, but mypy/ruf were complaining about that. Can I extend the line length in this PR ?